lesson_14/lesson_14_Turcu_Alexandru/challange_homework.py

```diff
@@ -0,0 +1,107 @@
+"""Design a game class that includes two genres of games and if you can play single or multi-player.
+
+The challenge must have the following criteria:
+
+1.Create at least two attributes for the sub classes and a overload method.
+2.You should be able to add revenues and compare two different genres and checkout score/rating by average/country.
+3.Create a Mixin class that displays the score of each game in order to know if it is worth playing.
+
+Solve the challenge"""
+
+from pythonlangutil.overload import Overload, signature
```

Comment:
from the challenge requirement, we need to overload just operators not user-defined function. So the __add__ is enough for operator overloading. The overloading that this library is making available is another type of overloading specific for other languages

Reply:
google was my friend ^_^